test/__snapshots__/prepend.spec.js.snap

```diff
@@ -13,10 +13,19 @@ exports[`prepend should override sets from options with CSS declared ones 1`] =
 }"
 `;
 
-exports[`prepend should prepend sets from options 1`] = `
+exports[`prepend should prepend sets from options for a object set 1`] = `
 ".dummy {
   color: tomato;
   font-size: 1.4rem;
   padding: 0 1rem;
 }"
 `;
+
+exports[`prepend should prepend sets from options for a string set 1`] = `
+".dummy {
+  fontSize: 1.4rem;
+          @media (width >= 500px) {
```

Comment:
Smth wrong with indentation

Reply:
hmm it's the Jest generated string, if I change the indentation here it breaks the test. If I change indentation in `test/prepend.spec.js` I can make it look pretty here but not there. Any ideas?

Reply:
Snapshots should not be edited for sure. We will see what we can do.